### PR TITLE
#161348786 (Integrate TravisCI):Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by dnuwa**
#### What does this PR do?
Integrate TravisCi with Authors Haven
#### Description of Task to be completed.
- Add a travis badge to the README file
- Add a travis.yml  with project tests configurations
#### How should this be manually tested.
- travis badge automatically shows the status of build build into the README
#### What are the relevant pivotal tracker stories?
[#161348786](https://www.pivotaltracker.com/story/show/161348786)

### CODE REVIEW

**Review by judeinno — COMMENTED** [review]
Good work, keep it up.

**Comment by brucemakallan** [comment]
Danny, please add the ID of the chore in the heading of this PR.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+script:
+    - python manage.py test
+    - coverage run manage.py test 
+    - coverage report -m
+    - coveralls
+    
+after_success:
+    - coveralls
+
```
Comment (emugaya) [inline .travis.yml]: Add new line

**Review by emugaya — CHANGES_REQUESTED** [review]
Resolve merge conflicts.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-[![Coverage Status](https://coveralls.io/repos/github/andela/ah-backend-thanos/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-backend-thanos?branch=develop)
+
```
Comment (emugaya) [inline README.md]: Why are you removing `coveralls`? Are you going to use `codeclimate`?

**Review by emugaya — CHANGES_REQUESTED** [review]
Address my comment

File: README.md (hunk 2 above)
Comment (dnuwa) [reply README.md]: You had already merged the pr that adds the coveralls file. so develop already has the coveralls file. this pr only adds the travis badge.

**Pull request merged**